CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - [git] removed exports from namespace `defaultGutterStyles`, `maxWidth`, `continuationStyle`, `highlightStyle`). [#9999](https://github.com/eclipse-theia/theia/pull/9999)
 - [task] `TaskDefinition.properties.required` is now optional to align with the specification [#10015](https://github.com/eclipse-theia/theia/pull/10015)
 - [core] `setTopPanelVisibily` renamed to `setTopPanelVisibility` [#10020](https://github.com/eclipse-theia/theia/pull/10020)
+- [core] add additional startup logging for plugin starting and application loading [#10116](https://github.com/eclipse-theia/theia/pull/10116) - Contributed on behalf of STMicroelectronics
```

Comment:
Since the majority of the changes are in the `plugin-ext` package, it might make sense to add that to the changelog?

Reply:
I rebased and changed the changelog message from `[core]` to `[plugin-ext]`. Or would you expect two different entries? One for the `plugin-ext` and one for the `core` changes?